actiontext/app/helpers/action_text/content_helper.rb

```diff
@@ -18,17 +18,31 @@ def sanitize_action_text_content(content)
       sanitizer.sanitize(content.to_html, tags: allowed_tags, attributes: allowed_attributes, scrubber: scrubber).html_safe
     end
 
+    def render_action_text_attachment(attachment, locals: {})
+      options = { locals: locals, object: attachment, partial: attachment }
+
+      if attachment.respond_to?(:to_attachable_partial_path)
+        options[:partial] = attachment.to_attachable_partial_path
+      end
+
+      if attachment.respond_to?(:model_name)
+        options[:as] = attachment.model_name.element
+      end
+
+      render(**options).chomp
+    end
```

Comment:
We generally like to keep methods in the order they’re called (“table-of-contents order”). This one would go below `render_action_text_attachments`.